staging/src/k8s.io/pod-security-admission/policy/check_runAsNonRoot.go

```diff
@@ -126,3 +133,12 @@ func runAsNonRoot_1_0(podMetadata *metav1.ObjectMeta, podSpec *corev1.PodSpec) C
 
 	return CheckResult{Allowed: true}
 }
+
+func runAsNonRoot_1_28(podMetadata *metav1.ObjectMeta, podSpec *corev1.PodSpec, userNamespacesEnabled bool) CheckResult {
+	// See KEP-127: https://github.com/kubernetes/enhancements/blob/308ba8d/keps/sig-node/127-user-namespaces/README.md?plain=1#L411-L447
+	if userNamespacesEnabled {
```

Comment:
shouldn't this only relax when `userNamespacesEnabled` AND the podSpec sets HostUsers to false?

Reply:
same comment applies to other checks

Reply:
Yes makes sense, I changed the function `isUserNamespacesEnabledForPod` to cover those checks.